app/javascript/legacy/campaigner_facing/shares_editor.js

```diff
@@ -165,10 +164,9 @@ ee.on('shares:edit', function() {
   setupOnce('.shares-editor', SharesEditor);
 });
 
-$(function() {
-  new Clipboard('.share-copy-url');
-
-  $('.share-copy-url').on('click', (e: JQueryEventObject) => {
+window.addEventListener('DOMContentLoaded', () => {
+  $('.share-copy-url').on('click', e => {
```

Comment:
Clipboard.js broke in production (an `i is not a function` type of error). It turned out that the update to clipboard.js v2 introduces breaking changes (and seems to only declare a global, `ClipboardJS`). The two options were to downgrade back to v1.x or swap it for `copy-text-to-clipboard`. I preferred the latter (0.2kb vs 3.2kb, every little counts!)